### PR TITLE
OCPBUGS#12160: Release note for "graph:true" option

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.12

Issue: https://issues.redhat.com/browse/OCPBUGS-12160

Link to docs preview: https://59052--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-graph-true-field-disconnected

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -210,6 +210,10 @@ For more information, see xref:../installing/installing_with_agent_based_install
 You can perform an Agent-based installation in a disconnected environment. To create an image that is used in a disconnected environment, the `imageContentSources` section in the `install-config.yaml` file must contain the mirror information or `registries.conf` file if you are using ZTP manifests. The actual configuration settings to use in these files are supplied by either the `oc adm release mirror` or `oc mirror` command.
 For more information, see xref:../installing/installing_with_agent_based_installer/understanding-disconnected-installation-mirroring.adoc#understanding-disconnected-installation-mirroring[Understanding disconnected installation mirroring].
 
+[id="ocp-4-12-graph-true-field-disconnected"]
+==== Explanation of a `graph: true` field when creating the image set configuration
```
Comment (skrthomas) [inline release_notes/ocp-4-12-release-notes.adoc:214]: Per the contributing guides, Do not use backticks or other markup in assembly or module headings. I verified with the peer review squad D and this also applies to release note headings. Maybe instead you could use "Explanation of field to build and push graph-data"?

**Review by skrthomas — COMMENTED** [review]
One small nit, otherwise LGTM :) Nice work!

File: release_notes/ocp-4-12-release-notes.adoc (hunk 1 above)
Comment (xenolinux) [reply release_notes/ocp-4-12-release-notes.adoc]: Agree and addressed. Thanks :)

**Comment by opayne1** [comment]
@xenolinux this PR actually needs to go through change management because you are adding to the release notes that have already been published. cc @JoeAldinger

File: release_notes/ocp-4-12-release-notes.adoc [hunk 2]
```diff
@@ -210,6 +210,10 @@ For more information, see xref:../installing/installing_with_agent_based_install
 You can perform an Agent-based installation in a disconnected environment. To create an image that is used in a disconnected environment, the `imageContentSources` section in the `install-config.yaml` file must contain the mirror information or `registries.conf` file if you are using ZTP manifests. The actual configuration settings to use in these files are supplied by either the `oc adm release mirror` or `oc mirror` command.
 For more information, see xref:../installing/installing_with_agent_based_installer/understanding-disconnected-installation-mirroring.adoc#understanding-disconnected-installation-mirroring[Understanding disconnected installation mirroring].
 
+[id="ocp-4-12-graph-true-field-disconnected"]
+==== Explanation of field to build and push graph-data
+When creating the image set configuration, you can add the `graph: true` field to build and push the graph-data image to the mirror registry. The graph-data image is required to create OpenShift Update Service (OSUS) applications. The `graph: true` field also generates the `UpdateService` CR manifest. The `oc` command-line interface (CLI) can use the `UpdateService` CR manifest to create OSUS applications.
```
Comment (LalatenduMohanty) [inline release_notes/ocp-4-12-release-notes.adoc:215]: We should replace `create OpenShift Update Service (OSUS) applications` with `create OpenShift Update Service (OSUS)`. 

Replace `CR` with `custom resource` and `OSUS applications` OSUS`.

**Review by LalatenduMohanty — CHANGES_REQUESTED** [review]

**Comment by xltian** [comment]
lgtm

**Review by LalatenduMohanty — APPROVED** [review]
/lgtm

**Comment by xenolinux** [comment]
yes information is captured in the docs via https://github.com/openshift/openshift-docs/pull/57989/files

**Comment by ahardin-rh** [comment]
LGTM

**Comment by wking** [comment]
> yes information is captured in the docs via https://github.com/openshift/openshift-docs/pull/57989/files

Hmm, should the wording there be updated to pick up @LalatenduMohanty 's [suggestions][1]?  Or is updating that wording being tracked elsewhere?

[1]: https://github.com/openshift/openshift-docs/pull/59052#discussion_r1184815912

**Comment by xenolinux** [comment]
> > yes information is captured in the docs via https://github.com/openshift/openshift-docs/pull/57989/files
> 
> Hmm, should the wording there be updated to pick up @LalatenduMohanty 's [suggestions](https://github.com/openshift/openshift-docs/pull/59052#discussion_r1184815912)? Or is updating that wording being tracked elsewhere?

Hi @wking I created a PR https://github.com/openshift/openshift-docs/pull/59714 to track the suggestion.

**Review by LalatenduMohanty — APPROVED** [review]
/lgtm

**Pull request merged**